internal/generator/vector/output/otlp/otlp_all.toml

```diff
@@ -0,0 +1,374 @@
+# Route container, journal, and audit logs separately
+[transforms.output_otel_collector_reroute]
+type = "route"
+inputs = ["pipeline_my_pipeline_viaq_0"]
+route.container = '.log_source == "container"'
+route.node = '.log_source == "node"'
+route.auditd = '.log_source == "auditd"'
+route.kubeapi = '.log_source == "kubeAPI"'
+route.openshiftapi = '.log_source == "openshiftAPI"'
+route.ovn = '.log_source == "ovn"'
+
+# Normalize container log records to OTLP semantic conventions
+[transforms.output_otel_collector_container]
+type = "remap"
+inputs = ["output_otel_collector_reroute.container"]
+source = '''
+  # Create base resource attributes
+  resource.attributes = []
+  resource.attributes = append( resource.attributes,
+      [{"key": "cluster.id", "value": {"stringValue": get!(.,["openshift","cluster_id"])}},
+  	{"key": "openshift.log.source", "value": {"stringValue": .log_source}}]
+  )
+  # Append container resource attributes
+  resource.attributes = append( resource.attributes,
+      [{"key": "k8s.pod.name", "value": {"stringValue": get!(.,["kubernetes","pod_name"])}},
+  	{"key": "k8s.pod.uid", "value": {"stringValue": get!(.,["kubernetes","pod_id"])}},
+  	{"key": "k8s.container.name", "value": {"stringValue": get!(.,["kubernetes","container_name"])}},
+  	{"key": "k8s.container.id", "value": {"stringValue": get!(.,["kubernetes","container_id"])}},
+  	{"key": "k8s.namespace.name", "value": {"stringValue": get!(.,["kubernetes","namespace_name"])}}]
+  )
+  # Append kube pod labels
+  if exists(.kubernetes.labels) {
+      for_each(object!(.kubernetes.labels)) -> |key,value| {
+  	    resource.attributes = append(resource.attributes,
+              [{"key": "k8s.pod.label." + key, "value": {"stringValue": value}}]
+  	    )
+      }
+  }
+  # Create logRecord object
+  r = {}
+  r.timeUnixNano = to_string(to_unix_timestamp(parse_timestamp!(.@timestamp, format:"%+"), unit:"nanoseconds"))
+  r.observedTimeUnixNano = to_string(to_unix_timestamp(now(), unit:"nanoseconds"))
+  # Convert syslog severity keyword to number, default to 9 (unknown)
+  r.severityNumber = to_syslog_severity(.level) ?? 9
+  r.body = {"stringValue": string!(.message)}
+  r.attributes = []
+  # Append logRecord attributes
+  r.attributes = append(r.attributes,
+      [{"key": "openshift.log.type", "value": {"stringValue": .log_type}}]
```

Comment:
Not unless we want to group/reduce by that as well.   All resource.attributes are squashed, but logRecord.attributes are unique.   Happy to discuss.

Reply:
In the context of container logs... this is unique to the resource and not the log records.  Same with all the other log types too:
* journal: group by: cluster, node
* audit: group by: cluster, source

log type should not factor into the grouping since its a constant within each of these groupings